src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -84,7 +79,7 @@ AccessibleRoot* AccessibleItem::accessibleRoot() const
     }
 
     RootItem* rootItem = m_element->explicitParent() ? score->rootItem() : score->dummy()->rootItem();
-    return dynamic_cast<AccessibleRoot*>(rootItem->accessible());
+    return dynamic_cast<AccessibleRoot*>(rootItem->accessible().get());
```

Comment:
why do we need dynamic_cast? (yes I know it was already there...)

Reply:
since accessible() is the parent, we need to return the child

Reply:
Ah ok, I misread - AccessibleRoot derives from AccesibleItem, which is a bit counter-intuitive.  Still, I probably would have defined RootItem to return an AccessibleRoot rather than do the case here.